src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import 'variables';
+@import 'product-card';
+@import 'stars';
+
```

Comment:
Missing declare: 

```
* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}
```

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
Thanks for the tip, I'll make the adjustments